client/payment-methods/index.js

```diff
@@ -302,6 +242,30 @@ const PaymentMethods = () => {
 					</CardHeader>
 				) }
 
+				{ ! isUpeEnabled && (
+					<CardHeader className="payment-methods__header">
+						<InlineNotice
+							icon
+							status="warning"
+							isDismissible={ false }
+						>
+							{ interpolateComponents( {
+								mixedString: __(
+									'You were rolled back from dUPE to legacy card. Stay tuned on next actions.' +
```

Comment:
The wording is currently being aligned with the Design Team. paJDYF-9xf-p2#comment-20974 I'll update the text once confirmed.